app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<% provide :title, t(".login_title") %>
+<h1><%= t ".heading" %></h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for :session, url: login_path do |f| %>
+      
+      <%= f.label :email %>
+      <%= f.email_field :email, class: "form-control" %>
+      
+      <%= f.label :password %>
+      <%= f.password_field :password, class: "form-control" %>
+
+      <%= f.submit t(".login"), class: "btn btn-primary" %>
+    <% end %>
+
+    <p><%= t ".btn_signup" %> <%= link_to t(".signup_now"), signup_path %></p>
```

Comment:
move into form, please

Reply:
sorry maybe i know what is “move into form" means?

Reply:
yesss, that's right

Reply:
fix this comment, please

Reply:
sorry, my fault, my ques is : "sorry may i know what is "move in to form" means?

Reply:
I think i find the way. Solved